### ISSUE TITLE
Accuracy Function

### ISSUE THREAD

**Issue by PaperKites** [naoto0804/pytorch-inpainting-with-partial-conv#39]
Hello

I'm wondering if you have used an accuracy function for determining the accuracy during the training because I couldn't find any metric function to tell how good is the output.

Also, does the code run over the whole dataset in each iteration (epoch) or does it pick 8 images randomly each iteration? (considering batch size=8)

Thank you in advance!

**Comment by naoto0804** [comment]
I haven't implemented but you can try some metrics like L1 or L2 error used in the following;
https://arxiv.org/pdf/1806.03589.pdf

You may refer to [here](https://pytorch.org/docs/stable/data.html#torch.utils.data.DataLoader) for understanding how to load the data

**Comment by PaperKites** [comment]
So, Simply the code reads a random batch of images and gives it to the model every epoch, right?
https://github.com/naoto0804/pytorch-inpainting-with-partial-conv/blob/e04c84d2bfc2ef8b40ba4af13b08ea43c6057ea8/train.py#L107-L111
